import "MsgHead.proto";    


message MsgFarmStateReq
{

}

message MsgSeedCellReq
{
	required int32 seedlevel = 1;
	required int32 cellid = 2;
}

message MsgSpreadManureReq
{
	required int32 manurelevel = 1;
	required int32 cellid = 2;
}

message MsgGatherPloughCellReq
{
	required int32 cellid = 1;
}

message MsgWaterCellReq
{
	required int32 cellid = 1;

}
message MsgBuildMillReq
{
	required int32 milllevel = 1;
}

message MsgBuildWaterWayReq
{
	required int32 waterwaylevel = 1;
}